src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -8,19 +8,23 @@
 using Mono.Cecil;
 using Mono.Cecil.Cil;
 using Mono.Cecil.Rocks;
+using System;
+using System.Xml.Linq;
 
 namespace Coverlet.Core.Instrumentation
 {
     internal class Instrumenter
     {
         private string _module;
         private string _identifier;
+        private IEnumerable<string> _excludedFiles;
         private InstrumenterResult _result;
 
-        public Instrumenter(string module, string identifier)
+        public Instrumenter(string module, string identifier, IEnumerable<string> excludedFiles = null)
         {
             _module = module;
             _identifier = identifier;
+            _excludedFiles = excludedFiles;
```

Comment:
if you change this to `_excludedFiles = excludedFiles ?? Enumerable.Empty<string>()` you won't need the null checks elsewhere

Reply:
+1